packages/builder/test/fixture-projects/project-with-scripts/successful-script.js

```diff
@@ -1,3 +1,9 @@
-if (global.config === undefined || global.network === undefined) {
-  process.exit(123123);
+
+const fs = require("fs");
+
+export default async function (runtimeEnv, deployer, accounts) {
+  if (runtimeEnv.config === undefined || runtimeEnv.network === undefined) {
+    throw new Error("Config was not provided");
+  }
+  fs.appendFileSync("output.txt", runtimeEnv.network.name);
```

Comment:
we can have one script with IO. This one is good.

Reply:
It's easier to ensure script execution order through IO.
I'm not sure what can be a different good way to do it.